### ISSUE TITLE
Cannot Use With Deno 1.5.1

### ISSUE THREAD

**Issue by ebebbington** [syumai/dinatra#77]
```typescript
import { App, get } from "https://denopkg.com/syumai/dinatra/mod.ts";

const app = new App(8000);

app.register(get("/", () => "Hello, Bench!"));

app.serve();
```
```
$ deno run -A file.ts

error: TS1205 [ERROR]: Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.
export { Response } from "./response.ts";
         ~~~~~~~~
    at https://raw.githubusercontent.com/syumai/dinatra/master/mod.ts:25:10
```

**Comment by ebebbington** [comment]
I assume it's because you're importing the `Response` from the same file twice in `mod.ts`

If you don't have time to fix it i can make a PR

**Comment by syumai** [comment]
@ebebbington I'm sorry for late reply, this was fixed in 0.14.0 :pray: 
https://github.com/syumai/dinatra/pull/78
